assets/js/custom.js

```diff
@@ -1,15 +1,100 @@
-var controllerHome = (function(jQuery) {
-    var adjustHomeLink = function() {
-        if (document.location.hostname === 'localhost') {
-            jQuery('.navbar-brand').attr('href', '/');
-            jQuery('#nav-0').attr('href', '/');
-        }
-    };
-
-    return {
-        init: adjustHomeLink
-    };
-
-}(jQuery));
+var controllerHome = (function (jQuery) {
+  const CONSENT = 'consentGiven';
+  const consentBanner = jQuery("#consent-banner");
```

Comment:
Usually we initialize jQuery variables with `$` 

```suggestion

  const $consentBanner = jQuery("#consent-banner");
```

Reply:
I checked `mentor.js` as you suggested and  I confirmed that `$` was not added to the variables,  reason why I did not include it @dricazenck

Reply:
I understand, but that was the intention.. of course is not mandatory. And some of the PR's it was not mentioned which is a convention for the project. Maybe this should be explicit in the README. But all good if you don't do it.. it is easy to read the code and know what is a simple variable and what is Jquery Object. This improves the readability and maintainability of the code.